### PR TITLE
macos-build.yml: disable PDAL to avoid build error (refs #39971)

### PR DESCRIPTION
**Pull request by rouault**

### CODE REVIEW

**Comment by rouault** [comment]
hum, why aren't the OSX tests run... ?

File: .github/workflows/macos-build.yml [hunk 1]
```diff
@@ -47,12 +47,13 @@ jobs:
           mkdir -p ../build-QGIS
           cd ../build-QGIS
 
+          # FIXME: PDAL disable because of #39971
```
Comment (m-kuhn) [inline .github/workflows/macos-build.yml:50]:
```suggestion
```

**Comment by m-kuhn** [comment]
Sorry for hijacking, tests were not successful

**Comment by rouault** [comment]
Wondering if someone wouldn't have manually disabled the osx action ? I can't access https://github.com/qgis/qgis/settings/actions

**Comment by m-kuhn** [comment]
It was run here https://github.com/qgis/QGIS/pull/39978 so can't be this

**Comment by rouault** [comment]
you wanted to remove the comment line, but you didn't finally ?

**Comment by m-kuhn** [comment]
I did and the test didn't appear, so I deleted the commit and force pushed again

**Comment by nyalldawson** [comment]
Unrelated test failure

**Comment by PeterPetrik** [comment]
this can be reverted, uploaded new qgis-deps-0.6.0 which should solve this (see https://github.com/qgis/QGIS-Mac-Packager/releases/tag/qgsi-deps-0.6.0-2)

**Pull request merged**